Automattic/wp-calypso
# Checklist: Add a new item for the WordPress mobile app

#### Changes proposed in this Pull Request

* This will add a new checklist item for encouraging the users to more use the
WordPress mobile app.

#### Testing instructions



* Apply D19444-code to your sandbox and point it to `public-api.wordpress.com`.
* Open your website's checklist on `/checklist/YOUR-SITE-URL`.
* A new task _"Get the WordPress app"_ should be at the bottom of the list.
* Click on its _"Do it!"_ button and you see the `/me/get-apps` page.
* Back to the checklist. Check if the new task is marked as done.

_Note_: The "Get the WordPress app" task is user-based, which means it will be checked on every your site when you've done it once.
